rpcclient/examples/bitcoincoreunixsocket/main.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2014-2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/btcsuite/btcd/rpcclient"
+)
+
+func main() {
+	// Connect to local bitcoin core RPC server using HTTP POST mode over a Unix Socket.
```

Comment:
nit: all code should be wrapped at 80 characters (while configuring your IDE/editor to count tabs as 8 spaces).